README.md

```diff
@@ -122,29 +111,22 @@ For simplicity's sake, I'll call the name of the executable `spasm` below. Run t
 
     spasm -E -A skin.asm TIBoySkn.8xv
 
-To build the rom generation tool, use the provided Visual Studio solution in the [tiboyce-romgen](tiboyce-romgen) directory,
-or theoretically you can build the source for any platform with your C compiler of choice.
+To build the rom generation tool, use the provided Visual Studio solution in the [tiboyce-romgen](tiboyce-romgen) directory, or theoretically you can build the source for any platform with your C compiler of choice.
 
 Issues/Bugs
 -----------
 Report issues/bugs to the issue tracker, found here:
 
-https://github.com/calc84maniac/tiboyce/issues
+<https://github.com/calc84maniac/tiboyce/issues>
 
 License
 -------
-    TI-Boy CE - a Game Boy emulator for the TI-84 Plus CE calculator family.
-    Copyright (C) 2018 Brendan Fletcher
 
-    This program is free software: you can redistribute it and/or modify
-    it under the terms of the GNU General Public License as published by
-    the Free Software Foundation, either version 3 of the License, or
-    (at your option) any later version.
+TI-Boy CE - a Game Boy emulator for the TI-84 Plus CE calculator family.
+Copyright © 2018 Brendan Fletcher
+
+This program is free software: you can redistribute it and/or modify it under the terms of the GNU General Public License as published by the Free Software Foundation, either version 3 of the License, or (at your option) any later version.
```

Comment:
Instead of lines 128-132, might as well just link LICENSE.md

Reply:
Also, should I just add 
```
Copyright © 2019 Brendan Fletcher
```
as it is only a few days away?

Reply:
Yeah good idea, but might as well write `2018-2019` then.

Reply:
> Actually, do you even want to link it? You can click on the badge at the top and it will take you to it so I think we should not link it at the bottom but add the notice. Thoughts?

I mean, sure, but it could be phrased as "It's GPL3 - for more details, read LICENSE.md".
Anyway, the decision is calc84's :P

Reply:
True I'll leave it as it like that and see what he thinks